MAINTAINERS.md

```diff
@@ -80,12 +82,14 @@ project.
 [John Fastabend]: https://github.com/jrfastab
 [Kornilios Kourtis]: https://github.com/kkourt
 [Laurent Bernaille]: https://github.com/lbernail
+[Louis De Los Santos]: https://github.com/ldelossa
```

Comment:
I sign off everything without spaces.

Reply:
Was unsure which to use between the GitHub and Slack versions, and picked Slack's 🙈

Reply:
No worries :)